Applications/FileIO/GocadIO/GocadAsciiReader.cpp

```diff
@@ -28,13 +28,6 @@ namespace Gocad
 {
 namespace GocadAsciiReader
 {
-enum class NodeType
-{
-    UNSPECIFIED,
-    VRTX,
-    PVRTX
-};
-
```

Comment:
Please leave this in. Seeing this, I've noticed I don't check what this is set to but it's necessary and I'll fix it in the source code.

Reply:
The issue is fixed in #2622, the enum and the checks below are now actually used.